server/Cargo.toml

```diff
@@ -4,11 +4,11 @@ version = "0.0.1"
 edition = "2021"
 
 [dependencies]
-anvil = {  path = "foundry/anvil" }
-anvil-core = { path = "foundry/anvil/core" }
-anvil-rpc = {  path = "foundry/anvil/rpc" }
-anvil-server = { path = "foundry/anvil/server" }
-foundry-evm = { path = "foundry/evm" }
+anvil = { git = "https://github.com/recursive-research/foundry" }
+anvil-core = { git = "https://github.com/recursive-research/foundry" }
+anvil-rpc = { git = "https://github.com/recursive-research/foundry" }
+anvil-server = { git = "https://github.com/recursive-research/foundry" }
+foundry-evm = { git = "https://github.com/recursive-research/foundry" }
```

Comment:
no, i looked into this (albeit briefly), and that doesn't work. but the rust toolchain will climb through all subpaths of the main repo to find the libraries with these names

Reply:
@kevinhalliday Yeah to elaborate a bit, when you try to do this, you get an error which is something along the lines of "conflicting source arguments". Kind of annoying if you were to use only one tiny module in a massive source repo, since I'm pretty sure that this way just clones the whole thing to `~/.cargo/git`.